lumicks/pylake/calibration.py

```diff
@@ -1,3 +1,218 @@
+import re
+from collections import UserDict
+
+
+def _read_from_bl_dict(bl_dict):
+    mapping = {
+        "bead_diameter": "Bead diameter (um)",
+        "rho_bead": "Bead density (Kg/m3)",
+        "rho_sample": "Fluid density (Kg/m3)",
+        "viscosity": "Viscosity (Pa*s)",
+        "temperature": "Temperature (C)",
+        "driving_frequency_guess": "Driving data frequency (Hz)",
+        "distance_to_surface": "Bead center height (um)",
+        # Fixed diode parameters (only available when diode was fixed)
+        "fixed_alpha": "Diode alpha",
+        "fixed_diode": "Diode frequency (Hz)",
+        # Only available for axial with active
+        "drag": "gamma_ex_lateral (kg/s)",
+    }
+
+    return {key_param: bl_dict.get(key_bl) for key_param, key_bl in mapping.items()}
```

Comment:
Yes, because they may not exist. `get` does not raise but returns `None` instead. Only a subset of these keys will be available. Initially, I considered having the option to always return the full list of possible arguments (populating it with defaults), but decided against it to see if anyone asks for it first.

For now, we could just omit the ones that aren't present at the door, then we also don't need that filter at the end.
```
return {
        key_param: bl_dict[key_bl] for key_param, key_bl in mapping.items() if key_bl in bl_dict
    }
```

Reply:
ah yeah, I forgot about the default `None` return. either is fine